packages/terra-application-navigation/src/header/_Header.jsx

```diff
@@ -0,0 +1,177 @@
+   */
+  onSelectExtensionItem: PropTypes.func,
+  /**
+   * A configuration object with information pertaining to the application's user.
+   */
+  userConfig: userConfigPropType,
+  /**
+   * A function to be executed upon selection of the Header's utility button.
+   */
+  onSelectUtilityButton: PropTypes.func,
+  /**
+   * A ref Object that will be provided with a reference to the Header's utility button.
+   */
+  utilityButtonPopupAnchorRef: PropTypes.shape({ current: PropTypes.instanceOf(Element) }),
+  /**
+   * A function to be executed upon selection of the 'Skip to Content' button.
+   */
+  onSelectSkipToContent: PropTypes.func,
+  /**
+   * Key/Value pairs associating a string key entry to a numerical notification count.
+   */
+  notifications: PropTypes.object,
+  /**
+   * @private
+   * The currently active breakpoint.
+   */
+  activeBreakpoint: PropTypes.string,
+  /**
+   * @private
+   * Object containing intl APIs.
+   */
+  intl: intlShape,
+};
+
+const Header = ({
+  titleConfig,
+  navigationItems,
+  navigationRenderFunction,
+  activeNavigationItemKey,
+  onSelectNavigationItem,
+  notifications,
+  extensionItems,
+  activeBreakpoint,
+  onSelectExtensionItem,
+  userConfig,
+  onSelectSkipToContent,
+  intl,
+  onSelectUtilityButton,
+  utilityButtonPopupAnchorRef,
+}) => {
+  function renderTitle() {
+    if (!titleConfig) {
+      return null;
+    }
+
+    if (titleConfig.element) {
+      return (
+        <div title={titleConfig.title}>
+          {titleConfig.element}
+        </div>
+      );
+    }
+
+    return (
+      <React.Fragment>
+        {titleConfig.headline && <div className={cx('headline')} title={titleConfig.headline}>{titleConfig.headline}</div>}
+        <div className={cx('title')} title={titleConfig.title}>{titleConfig.title}</div>
+        {titleConfig.subline && <div className={cx('subline')} title={titleConfig.subline}>{titleConfig.subline}</div>}
+      </React.Fragment>
+    );
+  }
+
+  function renderNavigationTabs() {
+    return (
+      <Tabs
+        navigationItems={navigationItems}
+        activeTabKey={activeNavigationItemKey}
+        onTabSelect={onSelectNavigationItem}
+        notifications={notifications}
+        navigationRenderFunction={navigationRenderFunction}
+      />
+    );
+  }
+
+  function renderExtensions() {
+    if (!extensionItems || !extensionItems.length) {
+      return null;
+    }
+
+    return (
+      <Extensions
+        extensionItems={extensionItems}
+        activeBreakpoint={activeBreakpoint}
+        onSelect={onSelectExtensionItem}
+        notifications={notifications}
+      />
+    );
+  }
+
+  function renderUtilities() {
+    return (
+      <UtilityMenuHeaderButton
+        userConfig={userConfig}
+        onClick={onSelectUtilityButton}
+        popupAnchorRef={utilityButtonPopupAnchorRef}
+      />
+    );
+  }
+
+  return (
+    <div className={cx('header')}>
```

Comment:
Let's update this to the following. You'll also have to change the closing div to a closing header tag.
```suggestion
    <header role="banner" className={cx('header')}>
```

Reply:
Updated.